code/game/machinery/computer/dna_console.dm

```diff
@@ -682,13 +682,13 @@
 				else
 					injectorready = world.time + INJECTOR_TIMEOUT * 0.1
 
-				if(scanner_occupant?.client)
-					var/c_typepath = generate_chromosome()
-					var/obj/item/chromosome/CM = new c_typepath (drop_location())
-					if(LAZYLEN(stored_chromosomes) < max_chromosomes)
-						CM.forceMove(src)
-						stored_chromosomes += CM
-						to_chat(usr,"<span class='notice'>[capitalize(CM.name)] added to storage.</span>")
+				//if(scanner_occupant?.client)
+					//var/c_typepath = generate_chromosome()
+					//var/obj/item/chromosome/CM = new c_typepath (drop_location())
+					//if(LAZYLEN(stored_chromosomes) < max_chromosomes)
+						//CM.forceMove(src)
+						//stored_chromosomes += CM
+						//to_chat(usr,"<span class='notice'>[capitalize(CM.name)] added to storage.</span>")
```

Comment:
Either fix this (Make it so each gene can only reward a chromosome once), or remove it entirely and add an alternative method of getting chromosomes (unless there already is one, not sure how chromosomes are obtained)